test-suite/TestSpec.hs

```diff
@@ -9,10 +9,12 @@ import Test.Hspec.QuickCheck
 import Test.QuickCheck
 
 import Text.Layout.Table
-import Text.Layout.Table.Cell (determineCuts, CutInfo(..), determineCutAction, CutAction(..), applyCutInfo, viewRange)
+import Text.Layout.Table.Cell (Cell(..), determineCuts, CutInfo(..), determineCutAction, CutAction(..), applyCutInfo, viewRange)
 import Text.Layout.Table.Spec.OccSpec
 import Text.Layout.Table.Primitives.Basic
+import Text.Layout.Table.Primitives.AlignInfo
 import Text.Layout.Table.Justify
+import Text.Layout.Table.Cell.MultiFormatted
```

Comment:
If it does not make too much trouble, please sort it. (E.g. select and `!sort` in `vim`.)